File: generators/java/templates/src/main/java/com/mycompany/controllers/HomeController.java
package <%= namespace %>.controllers;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.servlet.ModelAndView;

@Controller
public class HomeController {

	@RequestMapping(value="/")
	public ModelAndView index() {
		ModelAndView result = new ModelAndView("index");
		result.addObject("Title", "Home Page");
		return result;
	}
	
	@RequestMapping(value="/about")
	public ModelAndView about() {
		ModelAndView result = new ModelAndView("about");
		result.addObject("Title", "About");
		result.addObject("Message", "Your application description page.");
		return result;
	}
	
	@RequestMapping(value="/contact")
	public ModelAndView contact() {
		ModelAndView result = new ModelAndView("contact");
		result.addObject("Title", "Contact");
		result.addObject("Message", "Your contact page.");
		return result;
	}
}

File: generators/node/templates/src/routes/homeController.js
var express = require('express');
var utils = require('../utility');

// application insights
let client = utils.initAppInsights();
let key = process.env.APPINSIGHTS_INSTRUMENTATIONKEY;

module.exports = function (app) {
   var router = express.Router();

   /* GET home page. */
   router.get('/', function (req, res) {
      res.render('index', { title: 'Home Page', instrumentationKey: key });
   });

   router.get('/contact', function (req, res) {
      res.render('contact', { title: 'Contact', message: 'Your contact page.', instrumentationKey: key });
   });

   router.get('/about', function (req, res) {
      res.render('about', { title: 'About', message: 'Your application description page.', instrumentationKey: key });
   });

   // Mount route as "/"
   app.use('/', router);
};